### DISCUSSION TITLE
APNG (Animated PNG) should be allowed to be uploaded and supported

### DISCUSSION THREAD

**Discussion by wis** [community/community#127623] [Repositories]
### Select Topic Area

Product Feedback

### Body

[APNG ](https://en.wikipedia.org/wiki/APNG) files still do not have support in the issue editor (nor the issue/markdown renderer), the file upload/attach feature gives an error saying file type is not supported and lists the supported file types:
![We don’t support that file type. Try again with GIF, JPEG, JPG, MOV, MP4, PNG, SVG, WEBM, CPUPROFILE, CSV, DMP, DOCX, FODG, FODP, FODS, FODT, GZ, JSON, JSONC, LOG, MD, ODF, ODG, ODP, ODS, ODT, PATCH, PDF, PPTX, TGZ, TXT, XLS, XLSX or ZIP.](https://github.com/community/community/assets/9993663/ae1b25ba-d74c-47ec-b21a-535347af5413)

APNG files should also be supported in rendered markdown files (e.g. readme.md), as a more efficient and better looking alternative to GIF.

APNG is widely supported on the web, [according to caniuse.com](https://caniuse.com/?search=apng):
![caniuse.com APNG Animated PNG (APNG) Like animated GIFs, but allowing 24-bit colors and alpha transparency. Global	97.33% browser marketshare support](https://github.com/community/community/assets/9993663/78e302aa-1b13-4618-9ff7-49669cae4b27)

**Comment by phuclv90** [comment]
<img width="480" height="400" alt="elephant" src="https://github.com/user-attachments/assets/9c6ffebe-5091-40ca-8e90-561d24fb244b" />

It does support APNG. Did you upload a file with apng extension? It should be \*.png
